### PR TITLE
Small fixes, update copyright and documentation dates

### PR DESCRIPTION
**Pull request by alwey**
Small changes in data, comments and documentation.

### CODE REVIEW

**Comment by devsgit** [comment]
@alwey 
This is off-topic of this PR, but it's the only way I'm able to "contact" you :)
I'm with chess.com and we found a cutechess need.

I’m reaching out to see if you might be interested in doing a little bit of paid custom work on cutechess to support some needs we have :slightly_smiling_face: I would be happy to pay for it, and have it be made open source!

We would like to introduce a unique pairings mode, which we call “Reverse Pyramid”. This allows the pairings to “build” to the top seed. Example:

If you have 10 engines:

`#1`
`#2`
`#3`
`#4`
`#5`
`#6`
`#7`
`#8`
`#9`
`#10`

We would do pairings going UP, where the engines at the bottom would play every engine below it first, and then move on.

`#10` has no engines below it, so skip that.

`#9` would play `#10`.

`#8` would play `#10` then `#9`.

`#7` would play `#10`, then `#9`, then `#8`.

…

`#1` would play `#10`, `#9`, `#8`…. up to `#2`.

We would also want to decide how many games each one would play in their pairings, etc.

Let me know if you would be willing to implement this! Again, I’m happy to pay for it.
Contact me at `dan@chess.com` if you would like to talk further.
Cheers!

**Comment by alwey** [comment]
@devsgit : For various reasons I am not in a position to do payed work in this context. But I will have a look at the code because I find your suggestion interesting. If I see an elegant way to solve this, I will publish a pull request.  I cannot promise you to provide a solution and especially provide a solution within a defined amount of time.

Best regards!

**Comment by ilaripih** [comment]
@devsgit @alwey And I will happily help if/when I have the time.

File: projects/cli/src/main.cpp [hunk 1]
```diff
@@ -660,7 +660,7 @@ int main(int argc, char* argv[])
 		{
 			out << "cutechess-cli " << CUTECHESS_CLI_VERSION << endl;
 			out << "Using Qt version " << qVersion() << endl << endl;
-			out << "Copyright (C) 2008-2017 Ilari Pihlajisto and Arto Jonsson" << endl;
+			out << "Copyright (C) 2008-2018 Ilari Pihlajisto and Arto Jonsson" << endl;
```
Comment (ilaripih) [inline projects/cli/src/main.cpp]: Could you change "Ilari Pihlajisto and Arto Jonsson" to "Cute Chess authors"?

File: projects/gui/src/main.cpp [hunk 2]
```diff
@@ -55,7 +55,7 @@ int main(int argc, char* argv[])
 		{
 			out << "Cute Chess " << CUTECHESS_VERSION << endl;
 			out << "Using Qt version " << qVersion() << endl << endl;
-			out << "Copyright (C) 2008-2015 Ilari Pihlajisto and Arto Jonsson" << endl;
+			out << "Copyright (C) 2008-2018 Ilari Pihlajisto and Arto Jonsson" << endl;
```
Comment (ilaripih) [inline projects/gui/src/main.cpp]: Could you change "Ilari Pihlajisto and Arto Jonsson" to "Cute Chess authors"?

File: translations/cutechess_zh_CN.ts [hunk 3]
```diff
@@ -1771,8 +1771,8 @@
     </message>
     <message>
         <location filename="../projects/gui/src/mainwindow.cpp" line="956"/>
-        <source>Copyright 2008-2017 Ilari Pihlajisto and Arto Jonsson</source>
-        <translation>版权所有 2008-2017 Ilari Pihlajisto 与 Arto Jonsson</translation>
+        <source>Copyright 2008-2018 Ilari Pihlajisto and Arto Jonsson</source>
```
Comment (ilaripih) [inline translations/cutechess_zh_CN.ts]: Could you change "Ilari Pihlajisto and Arto Jonsson" to "Cute Chess authors"?

**Comment by alwey** [comment]
@devsgit @ilaripih  Please refer to PR 401 as a possible solution.

**Pull request merged**